tailcfg/tailcfg.go

```diff
@@ -289,6 +290,11 @@ type Node struct {
 	// peer or any of its subnets. Traffic originating from subnet routes will
 	// not be masqueraded (e.g. in case of --snat-subnet-routes).
 	SelfNodeV4MasqAddrForThisPeer netip.Addr `json:",omitempty"`
+
+	// IsWireGuardOnly indicates that this is a non-Tailscale WireGuard peer, it
+	// is not expected to speak Disco or DERP, and it must have Endpoints in
+	// order to be reachable.
+	IsWireGuardOnly bool `json:",omitempty"`
```

Comment:
Maybe leave a note saying that as of `2023-04-06 only the first parseable Endpoint is used`